### ISSUE TITLE
Inquiry about long context data during annealing stage

### ISSUE THREAD

**Issue by luppx** [RUC-GSAI/YuLan-Mini#9]
您好, 我正在参考这篇论文进行long context training. 但是我对论文Table 5中所列出的数据集有一些疑问, 希望能得到您的回复, 非常感谢.
1. MathInstruct指的是[MAmmoTHMathInstruct数据集](https://huggingface.co/datasets/TIGER-Lab/MathInstruct)吗?
2. Table 5中列出的两个MathInstruct是同一个数据集吗, 有什么区别?
![Image](https://github.com/user-attachments/assets/e5f3c650-ee48-4064-ac76-4532e6cd78e6)
3. AMPS数据集有[AMPS_mathematica](https://huggingface.co/datasets/XinyaoHu/AMPS_mathematica)和[AMPS_khan](https://huggingface.co/datasets/XinyaoHu/AMPS_khan), 这两个数据集都包含数学data, 请问Table 5中的AMPS具体是指哪个数据集? 还是指这两个数据集?
4. 请问Tulu v3具体是指哪个数据集?可以给出相应的下载链接吗?
5. 你们在论文第17页的页脚中给出的LeetCode 数据集的url为https://huggingface.co/datasets/greengerong/leetcode, 但是在github 仓库中给出的url为[leetcode-solution-python](https://huggingface.co/datasets/richardodliu/leetcode_solution_python), 请问Table 5中的LeetCode数据集具体是指哪个数据集?
6. 请问OSS-Instruct (seed: the-Stack-v2)具体是指哪个数据集?可以给出相应的下载链接吗?
7. 请问Camel-ai具体是指哪个数据集?可以给出相应的下载链接吗?
8. 论文Section 5.3 Long Context Training中你们有提到'We upample books and concatenated GitHub code texts [Liu et al., 2024b] as long context data to capture long-term dependencies, while using high-quality short texts to preserve short-text capabilities', 请问这里的long context data是指longwanjuan数据集中的Book和Github domain吗?![Image](https://github.com/user-attachments/assets/451427c4-ee8a-405b-a50b-5ae2b5f1f284)

Hi, I'm trying to refer to this paper for long context training. But I have some questions about the datasets listed in Table 5 of the paper. I hope to get your reply. Thank you very much.
1. Does MathInstruct refer to the [MAmmoTHMathInstruct](https://huggingface.co/datasets/TIGER-Lab/MathInstruct) dataset?
2. Are the two MathInstructs listed in Table 5 the same dataset? What is the difference?
![Image](https://github.com/user-attachments/assets/e5f3c650-ee48-4064-ac76-4532e6cd78e6)
3. AMPS dataset has [AMPS_mathematica](https://huggingface.co/datasets/XinyaoHu/AMPS_mathematica) and [AMPS_khan](https://huggingface.co/datasets/XinyaoHu/AMPS_khan). Both datasets contain mathematical data. Which dataset does AMPS in Table 5 refer to? Or does it refer to these two datasets?
4. Which dataset does Tulu v3 refer to? Can you give the corresponding URL?
5. The LeetCode you gave in the footer of page 17 of the paper The URL of the dataset is https://huggingface.co/datasets/greengerong/leetcode, but the URL given in the github repository is [leetcode-solution-python](https://huggingface.co/datasets/richardodliu/leetcode_solution_python). Which dataset does the LeetCode dataset in Table 5 refer to?
6. Which dataset does OSS-Instruct (seed: the-Stack-v2) refer to? Can you give the corresponding URL?
7. Which dataset does Camel-ai refer to? Can you give the corresponding URL?
8. In Section 5.3 Long Context Training of the paper, you mentioned 'We upample books and concatenated GitHub code texts [Liu et al., 2024b] as long context data to capture long-term dependencies, while using high-quality short texts to preserve short-text capabilities'. Does the long context data here refer to the Book and Github domain in the longwanjuan dataset?![Image](https://github.com/user-attachments/assets/451427c4-ee8a-405b-a50b-5ae2b5f1f284)

**Comment by huyiwen** [comment]
@luppx Thanks for checking our data.

> 1. Does MathInstruct refer to the [MAmmoTHMathInstruct](https://huggingface.co/datasets/TIGER-Lab/MathInstruct) dataset?
> 2. Are the two MathInstructs listed in Table 5 the same dataset? What is the difference?
> 3. AMPS dataset has [AMPS_mathematica](https://huggingface.co/datasets/XinyaoHu/AMPS_mathematica) and [AMPS_khan](https://huggingface.co/datasets/XinyaoHu/AMPS_khan). Both datasets contain mathematical data. Which dataset does AMPS in Table 5 refer to? Or does it refer to these two datasets?
> 4. Which dataset does Tulu v3 refer to? Can you give the corresponding URL?
> 5. The LeetCode you gave in the footer of page 17 of the paper The URL of the dataset is https://huggingface.co/datasets/greengerong/leetcode, but the URL given in the github repository is [leetcode-solution-python](https://huggingface.co/datasets/richardodliu/leetcode_solution_python). Which dataset does the LeetCode dataset in Table 5 refer to?
> 6. Which dataset does OSS-Instruct (seed: the-Stack-v2) refer to? Can you give the corresponding URL?
> 7. Which dataset does Camel-ai refer to? Can you give the corresponding URL?
> 8. In Section 5.3 Long Context Training of the paper, you mentioned 'We upample books and concatenated GitHub code texts [Liu et al., 2024b] as long context data to capture long-term dependencies, while using high-quality short texts to preserve short-text capabilities'. Does the long context data here refer to the Book and Github domain in the longwanjuan dataset?

Due to limited space, Tables 4 and 5 can only display example data. We have made the **complete dataset** for each phase publicly available at [here](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets), with phases [26](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets/data_mix/26_20241211_015209.json) and [27](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets/data_mix/27_20241213_051741.json) being the annealing stages.

> In addition, what does Curated in Table 5 mean? Does it refer to synthetic data?

"Curated" means that we directly use the dataset as it is, while "CoT" and "Long CoT" imply that we have re-synthesized the answers to the questions ourselves, which can be find at [here](https://huggingface.co/datasets/yulan-team/YuLan-Mini-Datasets)

**Comment by luppx** [comment]
> [@luppx](https://github.com/luppx) Thanks for checking our data.
> 
> > 1. Does MathInstruct refer to the [MAmmoTHMathInstruct](https://huggingface.co/datasets/TIGER-Lab/MathInstruct) dataset?
> > 2. Are the two MathInstructs listed in Table 5 the same dataset? What is the difference?
> > 3. AMPS dataset has [AMPS_mathematica](https://huggingface.co/datasets/XinyaoHu/AMPS_mathematica) and [AMPS_khan](https://huggingface.co/datasets/XinyaoHu/AMPS_khan). Both datasets contain mathematical data. Which dataset does AMPS in Table 5 refer to? Or does it refer to these two datasets?
> > 4. Which dataset does Tulu v3 refer to? Can you give the corresponding URL?
> > 5. The LeetCode you gave in the footer of page 17 of the paper The URL of the dataset is https://huggingface.co/datasets/greengerong/leetcode, but the URL given in the github repository is [leetcode-solution-python](https://huggingface.co/datasets/richardodliu/leetcode_solution_python). Which dataset does the LeetCode dataset in Table 5 refer to?
> > 6. Which dataset does OSS-Instruct (seed: the-Stack-v2) refer to? Can you give the corresponding URL?
> > 7. Which dataset does Camel-ai refer to? Can you give the corresponding URL?
> > 8. In Section 5.3 Long Context Training of the paper, you mentioned 'We upample books and concatenated GitHub code texts [Liu et al., 2024b] as long context data to capture long-term dependencies, while using high-quality short texts to preserve short-text capabilities'. Does the long context data here refer to the Book and Github domain in the longwanjuan dataset?
> 
> Due to limited space, Tables 4 and 5 can only display example data. We have made the **complete dataset** for each phase publicly available at [here](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets), with phases [26](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets/data_mix/26_20241211_015209.json) and [27](https://github.com/RUC-GSAI/YuLan-Mini/tree/main/pretrain/datasets/data_mix/27_20241213_051741.json) being the annealing stages.
> 
> > In addition, what does Curated in Table 5 mean? Does it refer to synthetic data?
> 
> "Curated" means that we directly use the dataset as it is, while "CoT" and "Long CoT" imply that we have re-synthesized the answers to the questions ourselves, which can be find at [here](https://huggingface.co/datasets/yulan-team/YuLan-Mini-Datasets)

Understood, thanks for your reply.

**Comment by huyiwen** [comment]
Hi, we have updated a tokenized version of annealing datasets [here](https://github.com/RUC-GSAI/YuLan-Mini/blob/main/pretrain/datasets/README.md).

But preparing the text datasets will take some more time (as our cluster subscription has expired)
